website/content/docs/commands/alloc/signal.mdx

```diff
@@ -20,6 +20,10 @@ This command accepts a single allocation ID and a task name. The task name must
 be part of the allocation and the task must be currently running. The task name
 is optional and if omitted every task in the allocation will be signaled.
 
+Task name may also be specified using the `-task`  option rather than a command 
+argument. If task name is given with both an argument and the `-task` option, 
+preference is given to the latter.
```

Comment:
Our [style guide](https://github.com/hashicorp/engineering-docs/blob/master/writing/style-guide.md#local-colloquialisms) recommends avoiding the construct "latter" vs "former".

```suggestion
Task name may also be specified using the `-task`  option rather than a command 
argument. If task name is given with both an argument and the `-task` option, 
preference is given to the `-task` option.
```

(Same for the other command docs)